Formula/python3.rb

```diff
@@ -314,7 +314,11 @@ def sitecustomize
   end
 
   def caveats
-    xy = (prefix/"Frameworks/Python.framework/Versions").children.first.basename.to_s
+    if prefix.exist?
+      xy = (prefix/"Frameworks/Python.framework/Versions").children.first.basename.to_s
+    else
+      xy = version.to_s.slice(/(3\.\d)/) || "3.6"
```

Comment:
This isn't going to be correct and may be missed when doing a version bump (which is why the original refactoring was done). Could we consider just omitting the relevant lines of the caveats instead?